metafacture-strings/src/main/java/org/metafacture/strings/LineRecorder.java

```diff
@@ -81,7 +81,9 @@ public void resetStream() {
 
     @Override
     public void closeStream() {
+        assert !isClosed();
```

Comment:
This is rather unusual. AFAICS, no other module does this (inside `closeStream()` that is).

Reply:
Do you really mean `assert !isClosed();` ? There are alternatives like `if (!isClosed) {`. Does it make any difference, semantically? Would it be better to use the `if` statement?